espresso-server/app/build.gradle.kts

```diff
@@ -141,7 +141,6 @@ dependencies {
         because("Android Compose support")
     }
     androidTestImplementation("androidx.compose.ui:ui-test-junit4:${composeVersion}") {
-        isTransitive = false
```

Comment:
not sure if the value should be set conditionally then based on the actual `composeVersion` provided